client/driver/docker.go

```diff
@@ -99,18 +99,6 @@ type DockerDriverConfig struct {
 	WorkDir          string              `mapstructure:"work_dir"`           // Working directory inside the container
 }
 
-// Validate validates a docker driver config
-func (c *DockerDriverConfig) Validate() error {
```

Comment:
No ImageName won't disappear after validation, unless we set it to `""` somewhere in the code.